### ISSUE TITLE
Uncaught Error: This TextEditor has been destroyed

### ISSUE THREAD

**Issue by mistguy** [shd101wyy/atom-markdown-katex#25]
[Enter steps to reproduce below:]

1. open one markdown file, another pane containing the preview opens.
2. click on the preview pane.
3. open another markdown file

**Atom Version**: 1.7.3
**System**: linux 4.5.3-1-ARCH
**Thrown From**: [atom-markdown-katex](https://github.com/shd101wyy/atom-markdown-katex) package, v0.5.8

### Stack Trace

Uncaught Error: This TextEditor has been destroyed

```
At /usr/share/atom/resources/app.asar/src/display-buffer.js:794

Error: This TextEditor has been destroyed
    at DisplayBuffer.module.exports.DisplayBuffer.screenPositionForBufferPosition (/usr/share/atom/resources/app.asar/src/display-buffer.js:794:15)
    at TextEditorMarker.module.exports.TextEditorMarker.getHeadScreenPosition (/usr/share/atom/resources/app.asar/src/text-editor-marker.js:179:33)
    at Cursor.module.exports.Cursor.getScreenPosition (/usr/share/atom/resources/app.asar/src/cursor.js:73:26)
    at TextEditor.module.exports.TextEditor.getCursorScreenPosition (/usr/share/atom/resources/app.asar/src/text-editor.js:1490:35)
    at setTimeout (/home/wangc/.atom/packages/atom-markdown-katex/lib/md-preview.js:83:45)
```

### Commands

```
     -1:08.6.0 editor:newline (atom-text-editor.editor.is-focused)
     -1:04.9.0 core:paste (atom-text-editor.editor.is-focused)
     -1:02.0 core:backspace (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "welcome",
      "markdown-preview"
    ]
  },
  "atom-markdown-katex": {
    "pdfUseGithub": false
  }
}
```

### Installed Packages

```coffee
# User
atom-markdown-katex, v0.5.8
autocomplete-python, v1.7.2
file-icons, v1.7.11
language-haskell, v1.7.15
language-latex, v0.6.1
language-pfm, v0.82.0
minimap, v4.23.5
minimap-bookmarks, v0.3.1

# Dev
No dev packages
```

**Comment by shd101wyy** [comment]
Should be fixed at version 0.5.9. 
Thanks !

**Comment by mistguy** [comment]
confirm fixed
